### ISSUE TITLE
act not found after install with a bash script

### ISSUE THREAD

**Issue by mscheltienne** [SanjulaGanepola/github-local-actions#88]
I wanted to try this extension today, and I got blocked at the first step: setting up the components.
My already installed `Docker Engine` was immediately found, but `nektos/act` was not.

I ran the bash script on the installation instructions of `act`:

```
curl --proto '=https' --tlsv1.2 -sSf https://raw.githubusercontent.com/nektos/act/master/install.sh | sudo bash
```

From the folder `/opt/act`, and added the created binary to my path. My shells run `act` correctly, i.e. from one of my repositories:

```
mscheltienne@D-1027100163:~/git/mscheltienne/mne-lsl$ act --list
INFO[0000] Using docker host 'unix:///var/run/docker.sock', and daemon socket 'unix:///var/run/docker.sock' 
Stage  Job ID                 Job name                                                            Workflow name  Workflow file  Events                                     
0      test-editable          test editable install                                               ci             ci.yaml        pull_request,push,workflow_dispatch,release
0      sdist                  create sdist                                                        ci             ci.yaml        pull_request,push,workflow_dispatch,release
0      get_commit_message     get commit message                                                  ci             ci.yaml        pull_request,push,workflow_dispatch,release
0      paper                  draft PDF                                                           joss           draft-pdf.yml  pull_request,push,workflow_dispatch        
0      sha256                 sha256                                                              pylsl          pylsl.yaml     workflow_dispatch,schedule                 
0      generate               generate                                                            stubs          stubs.yaml     pull_request,schedule,workflow_dispatch    
1      cibuildwheel_emulated  build wheels on ${{ matrix.os }} ${{ matrix.arch }}                 ci             ci.yaml        pull_request,push,workflow_dispatch,release
1      cibuildwheel           build wheels on ${{ matrix.os }} ${{ matrix.arch }}                 ci             ci.yaml        pull_request,push,workflow_dispatch,release
2      doc-build              build documentation                                                 ci             ci.yaml        pull_request,push,workflow_dispatch,release
2      test-pip-pre           test pip pre-release - py${{ matrix.python }}                       ci             ci.yaml        pull_request,push,workflow_dispatch,release
2      check                  run twine check                                                     ci             ci.yaml        release,pull_request,push,workflow_dispatch
2      test                   test wheels on ${{ matrix.os }} - py${{ matrix.python }}            ci             ci.yaml        pull_request,push,workflow_dispatch,release
2      test-mne-compat        test mne compat ${{ matrix.mne-version }} - py${{ matrix.python }}  ci             ci.yaml        pull_request,push,workflow_dispatch,release
3      doc-deploy-release     deploy stable documentation                                         ci             ci.yaml        pull_request,push,workflow_dispatch,release
3      doc-deploy-dev         deploy development documentation                                    ci             ci.yaml        release,pull_request,push,workflow_dispatch
4      publish                publish to PyPI                                                     ci             ci.yaml        push,workflow_dispatch,release,pull_request
```

But no matter, in the components panel of the extension, it says "Not installed".

**Comment by SanjulaGanepola** [comment]
Hello @mscheltienne. Thank you for sharing this issue. I will investigate to see what is wrong here. In the meantime, you should be able to still use the extension as normal. Can you have a try running a workflow?

**Comment by mscheltienne** [comment]
It's true that the workflow actually show up, so let's give it a shot.

```
 *  Executing task: set -o pipefail; gh act --workflows ".github/workflows/ci.yaml" --job "sdist" --secret-file "" --var-file "" --input-file "" --eventpath "" 2>&1 | tee "/home/mscheltienne/.config/Code/User/globalStorage/sanjulaganepola.github-local-actions/ci_create sdist #1 - 20241126_120841.log" 

/usr/bin/bash: line 1: gh: command not found
```

`gh` might be a good candidate for the `Components` section. But onced added:

```
 *  Executing task: set -o pipefail; gh act --workflows ".github/workflows/ci.yaml" --job "sdist" --secret-file "" --var-file "" --input-file "" --eventpath "" 2>&1 | tee "/home/mscheltienne/.config/Code/User/globalStorage/sanjulaganepola.github-local-actions/ci_create sdist #2 - 20241126_121029.log" 

unknown command "act" for "gh"

Did you mean this?
        api
        auth
        co

Usage:  gh <command> <subcommand> [flags]
```

The installation/setup instruction are unclear to me at the moment.

**Comment by SanjulaGanepola** [comment]
In the VS Code settings, you are able to modify the base act command here:

![image](https://github.com/user-attachments/assets/432314c8-100e-4155-8965-faeeb1acb402)

As it mentions, in most cases this will just be `act`. You would only use `gh act` if you selected the `GitHub CLI` installation method. In fact, selecting this option will automatically update this setting accordingly.

In your case, since you are trying to use the `Bash Script Installation` method, you can keep it as `act`. However, there does seem to be a bug with this installation method at the moment in relation to the `PATH`. For now, can you try one of the other installation methods until this is fixed?

**Comment by SanjulaGanepola** [comment]
> The installation/setup instruction are unclear to me at the moment.

I agree with this point as well. I will add more documentation once this issue has been resolved.

**Comment by mscheltienne** [comment]
Alright, then now it makes a lot more sense, and I think there is room for improvement to avoid this situation.

I started by trying to install `act` through VSCode with the github CLI, which failed since the github CLI was absent from my system. This did however modify the value `GitHub Local Actions: Act Command` to `gh act`.

Then, I installed `act` through the bash script; but unaware of this setting `GitHub Local Actions: Act Command`, I reported that `act` was not found when in fact the issue is that the `Act Command` was changed and was now erroneous. 

Changing `GitHub Local Actions: Act Command` to `act` now shows `nektos/act` as installed.

---------

I think the setting `GitHub Local Actions: Act Command` should *not* change when a github CLI install *fails*.
